users/models.py

```diff
@@ -0,0 +1,5 @@
+from django.contrib.auth.models import AbstractUser
+
+
+class User(AbstractUser):
+    pass
```

Comment:
Po co tak to jest zrobione? Co nam to daje, że zrobiłeś model który nic nie zmienia z tego co już było?

Reply:
Nie rozumiem pytania.
Chodzi o to żebym zrobiłem tak:

class User(models.Model):
    name = models.Charfield(max_length=50)
    last_name = models.Charfield(max_length=50)
    username = models.Charfield(max_length=50)
    password = models.Charfield(max_length=50)
    email = models.EmailField()
    
Bo z tego co rozumiem User(AbstractUser) automatycznie generuje te linije

Reply:
Po pierwsze to jaki cel jest tego? Chcecie tworzyć swojego Usera i nie będziecie korzystać z tego który jest w django? Bo wcześniej odniosłem wrażenie, że będzie ten z Django, ta propozycja niżej też nie jest ok, o robieniu swojego Usera trzeba chwilkę poczytać i zobaczyć z czym się wiąże, ale przede wszystkim to trzeba wiedzieć po co się coś robi, jaki to ma mieć cel